openapi.yaml

```diff
@@ -1,65 +1,117 @@
 openapi: "3.0.0"
 info:
   version: "1.0.0"
-  title: Service Name
+  title: Service Color
+
 paths:
-  /rest/checker:
+  /v4/qrcode/checker:
     get:
-      summary: route to check if the service is up.
+      summary: Route to check if the service is up.
       operationId: checker
       tags:
         - checker
       responses:
-        '200':
+        "200":
           description: Returning OK
           content:
-            text/html:
+            application/json:
               schema:
-                type: string
-                example: "OK"
-        '500':
-          description: Internal Error
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    description: Always true for 200 response
+                  message:
+                    type: string
+                example:
+                  success: true
+                  message: "OK"
+        "500":
+          description: unexpected error
           content:
-            text/html:
+            application/json:
               schema:
-                type: string
-                example: "The server encountered an internal error"
-
+                $ref: "#/components/schemas/error"
+                example:
+                  success: false
+                  error:
+                    code: 500
+                    message: "An internal error occured"
 
-  /rest/example:
+  /v4/color/{r},{g},{b},{filename}:
```

Comment:
I think there's a typo, shouldn't it be `/v4/color/{r},{g},{b}/{filename}`?

Reply:
thanks, I corrected that and also a copy / paste issue (changed qrcode with color a few lines above)